### PR TITLE
Logo updated to meshery from layer5

### PR DESCRIPTION
**Pull request by duckling69**
**Description**

This PR fixes issue  #568 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -17,8 +17,11 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 community! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery-istio/master/.github/welcome/Layer5-celebration.png)
-
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Meshery-darkmode-celebration.svg">
```
Comment (goldfishdolphin) [inline .github/config.yml:21]: @duckling69  Please have a look at the path <source media="(prefers-color-scheme: dark)"
srcset="https://raw.githubusercontent.com/meshery/meshery-istio/master/.github/welcome/Meshery-darkmode-celebration.svg">

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]

**Comment by duckling69** [comment]
@goldfishdolphin @thisiskaransgit done with all reviewed changes

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada:

**Review by goldfishdolphin — APPROVED** [review]
Good Job @duckling69! 
Thank you  for your contribution.

**Review by theBeginner86 — APPROVED** [review]

**Pull request merged**